web/src/components/contactForm.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ContactWrapper = styled.div`
+    ${tw`text-center`}
+`;
+
+const FormStyling = styled.form`
+    ${tw`flex flex-col justify-center items-center`}
+`;
+
+const StyledHeading = styled.h1`
+    ${tw`font-bold mb-8 relative text-4xl text-black-3 leading-tight`}
+`;
+const InputWrapper = styled.div`
+    ${tw`flex gap-1 w-1/3 flex-wrap`}
+`;
+
+const ButtonStyling = styled.button`
+    ${tw`cursor-pointer py-2 px-6 max-w-max mt-16 text-xl tracking-widest`}
+
+    background: #00A09B;
+    transition: background 500ms ease-out;
+
+    &:hover {
+        background: #027874;
```

Comment:
Can we use a tailwind class for this?

Reply:
@asasmith i could not get the hover to work for this. hover:bg-hoverGreen --after i added the hovergreen color in the config

Reply:
You might have to restart you dev server to pick up the config changes

Reply:
is this a new green or one that already exists in the color palette?

Reply:
@asasmith this was migrated over from the landing page styling. I found this in the existing styling

Reply:
Let's use tailwind classes wherever possible.